server/configs/application.properties

```
@@ -14,41 +14,50 @@ server.port=@@serverPort@@
 # HTTP-only port for servers that need to handle both HTTPS (configure via server.port and server.ssl above) and HTTP
 #context.httpPort=8080
 
-context.dataSourceName[0]=jdbc/labkeyDataSource
-context.driverClassName[0]=@@jdbcDriverClassName@@
-context.url[0]=@@jdbcURL@@
-context.username[0]=@@jdbcUser@@
-context.password[0]=@@jdbcPassword@@
-context.maxTotal[0]=50
-context.maxIdle[0]=10
-context.maxWaitMillis[0]=120000
-context.accessToUnderlyingConnectionAllowed[0]=true
-context.validationQuery[0]=SELECT 1
-
-#context.logQueries[0]=true
-#context.displayName[0]=Alternate Display Name
-
+# Database connections. All deployments need a labkeyDataSource as their primary database. Add additional external
+# data sources by specifying the required properties (at least driverClassName, url, username, and password)
+# with a prefix of context.resources.jdbc.<dataSourceName>.
+context.resources.jdbc.labkeyDataSource.type=javax.sql.DataSource
+context.resources.jdbc.labkeyDataSource.driverClassName=@@jdbcDriverClassName@@
+context.resources.jdbc.labkeyDataSource.url=@@jdbcURL@@
+context.resources.jdbc.labkeyDataSource.username=@@jdbcUser@@
+context.resources.jdbc.labkeyDataSource.password=@@jdbcPassword@@
+context.resources.jdbc.labkeyDataSource.maxTotal=50
+context.resources.jdbc.labkeyDataSource.maxIdle=10
+context.resources.jdbc.labkeyDataSource.maxWaitMillis=120000
+context.resources.jdbc.labkeyDataSource.accessToUnderlyingConnectionAllowed=true
+context.resources.jdbc.labkeyDataSource.validationQuery=SELECT 1
+#context.resources.jdbc.labkeyDataSource.logQueries=true
+#context.resources.jdbc.labkeyDataSource.displayName=Alternate Display Name
+
+#context.resources.jdbc.otherDataSource.driverClassName=@@extraJdbcDriverClassName@@
+#context.resources.jdbc.otherDataSource.url=@@extraJdbcUrl@@
+#context.resources.jdbc.otherDataSource.username=@@extraJdbcUsername@@
+#context.resources.jdbc.otherDataSource.password=@@extraJdbcPassword@@
```

The extra data source name is also parameterized
```suggestion
#context.resources.jdbc.@@extraJdbcDataSource@@.driverClassName=@@extraJdbcDriverClassName@@
#context.resources.jdbc.@@extraJdbcDataSource@@.url=@@extraJdbcUrl@@
#context.resources.jdbc.@@extraJdbcDataSource@@.username=@@extraJdbcUsername@@
#context.resources.jdbc.@@extraJdbcDataSource@@.password=@@extraJdbcPassword@@
```